src/prommis/solvent_extraction/solvent_extraction.py

```diff
@@ -120,7 +120,7 @@ def initialize_main_model(
 
         """
 
-        model.mscontactor.material_transfer_term.fix(1e-8)
+        model.mscontactor.material_transfer_term.fix(1e-10)
```

Comment:
Why was this changed?

Reply:
This change improves initialization by significantly reducing the number of warnings given about variables being set to variables outside of their bounds